example/rpc.js

```diff
@@ -1,3 +1,4 @@
+require('dotenv').load()
```

Comment:
not required here

Reply:
without it I was not able to make rpc calls correctly, 8332 was used intead of the 18332 in my .env for RPC port. Where is it getting the default 8332 from than?

Reply:
The default is probably from https://github.com/dcousens/yajrpc/blob/master/index.js#L20

But,  `process.ENV` variables should be filled in by `require('dotenv').load()` in the `example/index.js`,  you shouldn't need to do it twice.